package.json

```diff
@@ -19,6 +19,7 @@
     "test:e2e:ci": "cross-env URL=http://localhost:3001 run-p --race test:serve test:e2e",
     "analyze": "source-map-explorer build/static/js/main.*",
     "bundlesize": "bundlesize",
+    "bundle-report": "webpack-bundle-analyzer build/stats.json",
```

Comment:
is this a better bundle analyser than source-map-explorer? do we need both or would you recommend switching over to jus this one?

Reply:
I find it easier to look at `webpack-bundle-analyzer`, as it shows more info such as the gzipped size and such (it's possible that `source-map-explorer` does that too, I just don't know about it).

We only need one, I'd switch to `webpack-bundle-analyzer` if that's ok with you!

Reply:
@fsdiogo got for it 👍